spec/unit/views/components/sidebar_section_spec.rb

```diff
@@ -44,4 +44,25 @@
     end
   end
 
+  context 'with attributes_table for resource' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.